README.md

```diff
@@ -98,7 +98,7 @@ The Go CDK provides generic APIs for:
 *   Unstructured binary (blob) storage
 *   Publish/Subscribe (pubsub)
 *   Variables that change at runtime (runtimevar)
-*   Connecting to MySQL and PostgreSQL databases (mysql, postgres)
+*   Connecting to MariaDB/MySQL and PostgreSQL databases (mariadb, mysql, postgres)
```

Comment:
The things in parens are references to specific packages, there's no `mariadb` package (I assume you use the `mysql` one?).

How about

* Connecting to MySQL (including MariaDB) and PostgreSQL database (mysql, postgres)

?

Reply:
@vangent Sure, I'm fine with that, thanks.